src/components/results/dynamicsimulation/dynamic-simulation-result-tab.js

```diff
@@ -52,28 +56,33 @@ const DynamicSimulationResultTab = ({ studyUuid, nodeUuid }) => {
                     />
                 </Tabs>
             </Box>
-            <Box>
+            <Box sx={styles.resultContainer}>
                 <TabPanelLazy
                     key={TAB_INDEX_TIME_SERIES}
                     selected={tabIndex === TAB_INDEX_TIME_SERIES}
                 >
-                    {dynamicSimulationResultPresent && (
-                        <DynamicSimulationResultTimeSeries
-                            studyUuid={studyUuid}
-                            nodeUuid={nodeUuid}
-                        />
-                    )}
+                    <DynamicSimulationResultTimeSeries
+                        studyUuid={studyUuid}
+                        nodeUuid={nodeUuid}
+                    />
+                </TabPanelLazy>
+                <TabPanelLazy
+                    key={TAB_INDEX_TIME_SERIES}
+                    selected={tabIndex === TAB_INDEX_TIME_LINE}
```

Comment:
Shouldn't the key be TAB_INDEX_TIME_LINE ?

Reply:
YES! DONE! by https://github.com/gridsuite/gridstudy-app/pull/1942/commits/c00389c26b2bdea92c2e196a289e9ffc62d1d2ea